frappe/website/doctype/personal_data_delete_request/personal_data_delete_request.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from frappe.model.document import Document
+from frappe.utils.verified_command import get_signed_params, verify_request
+
+class PersonalDataDeleteRequest(Document):
+
+	def after_insert(self):
+		if frappe.session.user in ['Administrator', 'Guest']:
```

Comment:
we must allow administrator to create a new request on someone's behalf. make a separate "email / username" field

Reply:
The administrator can now request deletion for any user via email.